Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -192,8 +192,9 @@ define([
         if (!defined(container)) {
             throw new DeveloperError('container is required.');
         }
-        if (defined(options) && defined(options.globe) && options.globe === false && options.imageryProvider !== false) {
-            throw new DeveloperError('To turn off the globe, options.imageryProvider must be set to false.');
+        if (defined(options) && defined(options.globe) && options.globe === false &&
+                defined(options.imageryProvider) && options.imageryProvider !== false) {
```

Comment:
I'm not crazy about the working on the error but I would prefer that we don't even do this check, if globe is off just silently ignore the other stuff.  Otherwise there are a host of other options that need to be checked: `selectedImageryProviderViewModel`, `imageryProviderViewModels`, `selectedTerrainProviderViewModel`, `terrainProviderViewModels`, and `terrainProvider`, possible more.